### ISSUE TITLE
Ask for help

### ISSUE THREAD

**Issue by DragsonCZ** [RafaelDeJongh/cap#33]
Hi,
In last update you added SGC Screens. I absolutely have no clue how to use them!  Can someone help me?    What I have to connect to what wiht wiremod?

**Comment by AlexALX** [comment]
I'm sorry for lack of documentation for it yet, its planned to write keyboard legend and how stuff works on cap wiki, but unfortunately i'm lack of time to do this in near future.

**Comment by Tyenkrovy** [comment]
If this is not proper protocol, I apologize. I thought it appropriate to bring it up in an already extant, relevant thread.

The SGC screens seem to have stopped working correctly, in that they now ignore the pressing of the enter key. I've tried uninstalling and reinstalling GMod repeatedly, including completely deleting my entire GMod folder, to no avail. Given that pressing enter is required to begin dialing an address or change the monitor to another screen, you can imagine this puts a bit of a damper on their use.

**Comment by Prawneen** [comment]
The only issue I have with the SGC Screens Stool, No matter what I do or put in. Even if it's the correct password, the self-destruct code is apparently always Invalid. And I have set a simple password like 4545, and entered it via the SGC screen. Always tells me it's invalid. I do have a Naquada bomb around as well. 

Sadly I have no idea what the issue is or if I am actually doing something wrong.

**Comment by ponchoandlefty** [comment]
My M.A.L.P goes through a stargate and it floats. It also won't go through th portal if it has any parts on it (thrusters, users ect.) Please help me thanks :)

**Comment by AlexALX** [comment]
1. Self-descruct code is different that cancel code.
2. All changes what you doing in codes manager is only applies to new server (need to respawn it).

If this not help then idk.

**Comment by Prawneen** [comment]
Mhmm. Odd. I tried it with the simple Self-Destruct code of 12345678. I ticked off the Auto-Connect for the Naquada bomb. Saved the codes. Spawned the Server, spawned a naquada bomb, slammed it against the server to have it connect. And then spawned me a screen and went to the, Gate data monitor. Typed in the password. Yet still nothing happens. I've tried the same, with my Naquada bomb having the exact detonation code. And once without having any code for the Naquada bomb. Nothing works sadly.

**Comment by AlexALX** [comment]
wait, where you placing your code? You should place it inside SGC screen tool code manager not naquadah bomb!

**Comment by Prawneen** [comment]
I did place the code inside the SGC Screen tool code manager. Saved it. I spawned a Naquadah bomb. Spawned the SGC Screen server. And a screen. Tried to put in the self-destruct code, still shows that it is not valid.

I'm honestly out of ideas by now.

**Comment by AlexALX** [comment]
Its showing exaclt not valid or nothing do? If second then you need to press f1 (or enter) after entered code to start self-descrution.

**Comment by Prawneen** [comment]
It is exactly showing, "Entered code is not valid"

**Comment by ponchoandlefty** [comment]
Is there a way to add effects to the stargate like when you go through it takes you about 5 seconds to get out. Or when you go through if you see the effects of going through the stargate rather than just instantly going through to the other side.

**Comment by AlexALX** [comment]
@ponchoandlefty
This is issue tracker, its not for requests. Also if you read description in steam workshop you will understand that we don't accept requests anymore, only bugfixes for gmod/wiremod updates.

Anyway if talk about your request - its possible, but it have too much problems and thats why never done. However, there is a secret address 9 chevron which has this option only for players (items move instantly). More info you can get by read ancient texts on arthur mantle (secret code wire input) and by watching this movie (at the end) https://youtu.be/M6ZckYlVZv4?t=310

**Comment by ponchoandlefty** [comment]
I have my SGC screen working but I cannot get the self-destruct code to work.

**Comment by AlexALX** [comment]
@ponchoandlefty @Prawneen

It seems like there is indeed issue with codes in mp (works in sp). As workaround for now you can hold C and press right mouse click on SGC Server, then those code manager, type codes have and save. This should work.

**Comment by ponchoandlefty** [comment]
It put in the code and it worked this time but now it just say standing by. Nothing happens could you help me?

**Comment by AlexALX** [comment]
Like i said before in this thread after "stand by" shows you need press f1 again.

**Comment by ponchoandlefty** [comment]
Thanks!

**Comment by ponchoandlefty** [comment]
It's not letting me aboard when I press f1 again.

**Comment by johnoldiges** [comment]
Hello, I know this probably isn't the best place to post this but I saw no request section and did not want to create another thread. Would it be possible to add a second stool mirroring this tool but make it usable with the Atlantis screens?

**Comment by AlexALX** [comment]
@johnoldiges sure, its possible, if someone will do lua code, materials, sounds, and then create pull request.

**Comment by joeyc773** [comment]
The main thing I don't understand is the part about a server. What does it mean by that? I think if I knew that part then I might be able to figure out the rest :)

**Comment by johnoldiges** [comment]
Right click to place the server while holding the stool. Then click "use" (default E) to turn on the server and the screens will initialize and auto connect to the nearest stargate.

**Comment by danc77** [comment]
I would like to know how to use the galaxy map screen and also how to dial 8/9 chevron addresses manually from the dialing computer, aka not through the address book screen.

**Comment by CryptAlchemy** [comment]
Here's how you use them.
** Make sure to spawn your screens with a keyboard otherwise you'll have a fun time trying to attach a keyboard to them xd
Place your screens using the tool by left-clicking.
Place your SGC computer using the tool by right-clicking.
Hold C and right-click on your SGC computer to set your codes.
To turn the system on, press e on the computer.
To turn the system off, hold e on the computer.
Press numpad 0 to switch between screens and press enter to select the screen after highlighting it using arrow keys.
To self destruct, go to the Gate Active screen and hold numpad 1; type in your code; press enter; press numpad 1 to activate the bomb.
To abort, go to the Gate Active screen and type your abort code only. Then press enter.
To dial, go to your address book screen and select one of the addresses with arrow keys and hit enter twice.
To manually dial, enter your desired address into the dialing computer screen.
To turn your screens on and off individually, press e on them.
To shut the gate down, press backspace on the dialing computer screen.

There's prolly more but that's all I can think of right now

**Comment by YouCa2016** [comment]
I have a problem with the Goa'uld Warrior: when I spawn any, they don't shoot at all. I've already reinstalled both the game and addon pack, but I'm still getting the same issue.

**Comment by xESCOx** [comment]
what are the strings to change the horizon type via wire, i havent found them going through the code. (probably looking in wrong place)

**Comment by ponchoandlefty** [comment]
Hey, I have a problem with the SGC screens. The screen says no signal on it but when I put the computer tower next to it the screen just goes black. I've tried everything I know of to get it turned on and it still won't work.

**Comment by CryptAlchemy** [comment]
@ponchoandlefty  This is a coding issue that I noticed. I made my own version of these screens with a fix for my server. I'll reference the CAP team to tell them how to fix it.
@AlexALX @RafaelDeJongh You guys should use NW instead of NW2 since NW2 doesn't always work; I compared both NW and NW2 for the CAP screens and NW worked much better. I have no clue why NW2 doesn't always work, but it's true so lol
